index.html

```diff
@@ -0,0 +1,148 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>UI test</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <header class="header">
+    <div class="header__container">
+      <div class="header__content">
+        <button id="btn" class="button button-radius text-grey bg-white">
+          Boost’in
+        </button>
+        <div class="header__title title">Hello, {{name}} 
+        </div>
+        <div class="header__text text text-grey">Want to accelerate Artur's benefits? <br>
+          Artur's advertising option allows you to benefit from Google and Facebook <br> technologies, managed by
+          experts, in
+          order to win new customers</div>
+        <button class="header__button button bg-blue text-white">Contact my coach</button>
+      </div>
+      <img class="header__devices" src="img/header/mockups.png" alt="">
+    </div>
+
+
+    <div class="header-cards">
+      <div class="header-card">
+        <div class="header-card__image">
+          <img src="img/header/icon/01.png" alt="">
+        </div>
+        <div class="header-card__title">
+          Grow your <br>
+          community
+        </div>
+        <div class="header-card__text text text-grey">
+          Build your page's credibility with new Facebook fans
+        </div>
+      </div>
+      <div class="header-card">
+        <div class="header-card__image">
+          <img src="img/header/icon/02.png" alt="">
+        </div>
+        <div class="header-card__title">
+          Increase <br>
+          your interactions
+        </div>
+        <div class="header-card__text text text-grey">
+          Build your page's credibility with new Facebook fans
+        </div>
+      </div>
+      <div class="header-card">
+        <div class="header-card__image">
+          <img src="img/header/icon/03.png" alt="">
+        </div>
+        <div class="header-card__title">
+          Boost your <br>
+          website traffic
+        </div>
+        <div class="header-card__text text text-grey">
+          Build your page's credibility with new Facebook fans
+        </div>
+      </div>
+      <div class="header-card">
+        <div class="header-card__image">
+          <img src="img/header/icon/04.png" alt="">
+        </div>
+        <div class="header-card__title">
+          Generate <br>
+          prospects
+        </div>
+        <div class="header-card__text text text-grey">
+          Build your page's credibility with new Facebook fans
+        </div>
+      </div>
+    </div>
+
+  </header>
+  <main class="main">
+    <div class="main__container">
+      <h2 class="main__title title">Why choose Artur <br>
+        for your advertising</h2>
+
+      <div class="main-card">
+        <div class="main-card__content">
+          <div class="subtitle">A team of experts <br>
```

Comment:
this creates line break forever, but technically it should be there only if content doesn't fit